vowpalwabbit/parser.cc

```diff
@@ -541,21 +541,31 @@ void enable_sources(vw& all, bool quiet, size_t passes, input_options& input_opt
 
       if (input_options.json || input_options.dsjson)
       {
+        auto mode = json_parser_mode::standard;
+        if (all.label_type == label_type::cb)
```

Comment:
Where are we at in trying to project cb into ccb automatically?

Reply:
This is done in the json parser. Only JSON can be projected